pkgs/applications/window-managers/dwl/default.nix

```diff
@@ -12,17 +12,14 @@
 , wayland-scanner
 , wayland
 , wayland-protocols
-, wlroots_0_16
+, wlroots
```

Comment:
We decided (IIRC) that we want to keep explicitly referencing wlroots versions (i.e., `wlroots_0_*` instead of `wlroots`) since there is no stable wlroots release yet and each new version comes with breaking changes so the packages do actually (almost) all depend on a specific `wlroots` version.
You can move this part to `all-packages.nix` though (probably preferable?), if you want ;)

Reply:
I will do this later for my own expressions.